katello/katello-backup

```diff
@@ -46,9 +46,9 @@ optparse = OptionParser.new do |opts|
   @dir = ARGV[0].dup
 end
 
-def run_cmd(command)
+def run_cmd(command, exit_codes=[0])
   `#{command}`
-  if $?.exitstatus != 0
+  if not exit_codes.include?($?.exitstatus)
```

Comment:
Let's do `unless exit_codes.include?($?.exitstatus)` rather than `if not`. Thanks @jcpunk I don't think it's too bad, I had drawn up something similar.